server/common/model/src/main/resources/proto/DeltaReceivedDataBodyPb.proto

```
@@ -0,0 +1,14 @@
+syntax = "proto3";
+option java_package = "com.alipay.sofa.registry.common.model.client.pb";
+option java_multiple_files = true;
+option go_package = "proto";
+
+import "PushResourcesPb.proto";
```

Buf (1.54.0)

import "PushResourcesPb.proto": file does not exist

(COMPILE)